aws/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-west-1"
+}
+
+variable "deployment_name" {
+  description = "Name for this deployment"
+  type        = string
+  default     = "sourcegraph-cluster"
+}
+
+### Node Group One ###
+variable "node_group_one_instance_types" {
+  description = "Instance types to use in node group two"
+  type        = list
+  default     = ["m5.4xlarge"]
```

Comment:
```suggestion
  default     = ["m6a.4xlarge"]
```

I think [m6a](https://aws.amazon.com/ec2/instance-types/m6a/) is cheaper with better performance

![image](https://user-images.githubusercontent.com/68532117/213617171-cf21dc2c-19a1-4a04-933d-d6a3c3ff2812.png)

![image](https://user-images.githubusercontent.com/68532117/213617102-5d7f82bf-8d70-4617-8c5b-644854f90d5f.png)